source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRBottomSheetPresentationController.mm

```diff
@@ -11,25 +11,43 @@
 @interface ACRBottomSheetPresentationController ()
 
 @property (nonatomic, strong) UIView *dimmingView;
+@property (nonatomic) CGFloat keyboardHeight;
+@property (nonatomic) BOOL isAdjustingForKeyboard;
 
 @end
 
 @implementation ACRBottomSheetPresentationController
 
+#pragma mark - Frame Calculation
+
 - (CGRect) frameOfPresentedViewInContainerView
 {
     CGFloat targetHeight = self.presentedViewController.preferredContentSize.height;
-    return CGRectMake(0, self.containerView.bounds.size.height - targetHeight, self.containerView.bounds.size.width, targetHeight);
+    CGFloat containerHeight = self.containerView.bounds.size.height;
+    CGFloat visibleHeight = containerHeight - self.keyboardHeight;
+    CGFloat y = visibleHeight - targetHeight;
+    y = y < 0 ? 0 : y;
+
+    return CGRectMake(0, y, self.containerView.bounds.size.width, targetHeight);
```

Comment:
Make sure we adjust height as well as the y axis. 
This will ensure that we get the scrollview's default behaviour of bringing the first responders in visible interactable area above keyboard and there is enough room